packages/app/hooks/use-claim-nft.ts

```diff
@@ -253,15 +253,15 @@ export const useClaimNFT = (edition: IEdition) => {
           "Oops. An error occured.",
           "We are currently experiencing a lot of usage. Please try again in one hour!"
         );
-      } else {
```

Comment:
moving it outside of else as we'll update the snackbar in any case after an error